easybuild/easyconfigs/g/genmod/genmod-3.7.4-foss-2018b-Python-3.9.1.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'PythonPackage'
+
+name = 'genmod'
+version = '3.7.4'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://moonso.github.io/genmod/'
+description = 'Annotating and analyzing primarily genetic models in the VCF file format.'
+
+toolchain = {'name': 'foss', 'version': '2018b'}
```

Comment:
Unless you need OpenMPI, FFTW, etc. it is better to not include unnecessary dependencies and stick with the minimal GCCcore/7.3.0 toolchain from Python/3.9.1-GCCcore-7.3.0-bare.